shared/src/common/types/preferences/wishlist-response-dto.type.ts

```diff
@@ -14,6 +14,7 @@ interface WishlistsResponseDto {
 type WishlistInput = {
   modelId?: string;
   complectationId?: string;
+  createdAt: string;
```

Comment:
I think `WishlistInput` shouldn't have the required `createdAt` property. It should be optional.